src/MarketingPageAcceptanceTests.Steps/Utils/UITest.cs

```diff
@@ -36,17 +37,49 @@ public UITest()
 
         public string DetermineUser(string scenarioTitle)
         {
-            return scenarioTitle.Contains("authority", System.StringComparison.OrdinalIgnoreCase) ? "authoriy" : "supplier";
+            return scenarioTitle.Contains("authority", StringComparison.OrdinalIgnoreCase) ? "authoriy" : "supplier";
         }
 
-        public void SetUrl(string solutionId, string userType = "supplier")
+        public void SetUrl(string solutionId = null, string userType = null)
```

Comment:
id null?

Reply:
The null value check just below this takes the solutionId string param, checks if it's null. If it is null, it uses the generated solution.Id created at the start of the test run, otherwise uses the provided value from the parameter